.travis.yml

```diff
@@ -16,15 +16,22 @@ virtualenv:
   system_site_packages: true
 
 env:
-  matrix:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" DOCS="1"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
   global:
-    secure: "HLfds+sQG4JcM52D7b/sTemGvBlVNdfGc4XvJ+py2rx25F4BWojpbOYx44nSiUjBO9U6Eo5kwiK0mWEXoPGg6trQcbiur9kgHrV3+hydlUUKe2/P/mnoP2JAxtu0JBSLxNcG40FdIwEE3Sd0w/ocmxMOp3RmOOVacAY48eWYeh8="
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+    # travis for docs build
+  - secure: "HLfds+sQG4JcM52D7b/sTemGvBlVNdfGc4XvJ+py2rx25F4BWojpbOYx44nSiUjBO9U6Eo5kwiK0mWEXoPGg6trQcbiur9kgHrV3+hydlUUKe2/P/mnoP2JAxtu0JBSLxNcG40FdIwEE3Sd0w/ocmxMOp3RmOOVacAY48eWYeh8="
+    # transifex
+  - secure: "FNbQORNKr+5UPf/34B4Glr+VmYtmZReWt0FVTwJ+sx1OWm1VbQvjpYhiqUiMxWtBEdVl238fqLSB1hno7lbHMxLEGUiXL9b4ZFhAxRVHigMtiN35NZ1Qy+BX0p44BzzRhiQOoOA192Z6q0Wjaz+4M4memstKOJ93TLE+PqQ249g="
+
+  matrix:
```

Comment:
There is already 2 spaces:

`^^matrix:`

It might be something else.
I'll keep this closer to https://github.com/OCA/connector/blob/9.0/.travis.yml